recipe/run_test.bat

```diff
@@ -0,0 +1,16 @@
+@echo on
+
+set "LIBCIFPP_DATA_DIR=%PREFIX%\share\libcifpp"
```

Comment:
Why does `LIBCIFPP_DATA_DIR` need to be set here, i.e. shouldn't it be automatically set by the activation script?

Reply:
I think it is not needed.